src/api/posts/test/posts.test.js

```diff
@@ -69,6 +69,19 @@ describe('/posts', () => {
     expect(res.body.length).toBeGreaterThan(0);
   });
 
+  test('request posts with a valid expand query param', async () => {
```

Comment:
Please add a test for `expand=1` as well.

Reply:
I tried to cover this scenario in my test but it either returns empty data or timeout, although the API works fine if I were requesting with my browser or Postman. Got any clue what might cause this problem?

Reply:
Debug it, add logging, etc.  You should have 150 posts created before this test runs.  We can't land this change without a test that proves it works.